### PR TITLE
Deleted unecessary namespaces existence check

### PR DESCRIPTION
**Pull request by ecfm**
https://issues.cask.co/browse/CDAP-7206
http://builds.cask.co/browse/CDAP-DUT4765-2

### CODE REVIEW

File: cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java [hunk 1]
```diff
@@ -157,20 +157,10 @@ public Boolean call() throws Exception {
...
-
-        if (namespaces.size() == 0) {
-          return false;
-        }
-        if (namespaces.contains(NamespaceMeta.DEFAULT)) {
-          return true;
-        }
-        throw new IllegalStateException("Default namespace not found. Instead found unexpected namespaces: "
-                                          + namespaces);
+        // list() method will throw exception if the http request receives response status HTTP_OK. If list() does
```
Comment (anwar6953) [inline cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java]: the first statement seems wrong

File: cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java [hunk 2]
```diff
@@ -157,20 +157,10 @@ public Boolean call() throws Exception {
...
-
-        if (namespaces.size() == 0) {
-          return false;
-        }
-        if (namespaces.contains(NamespaceMeta.DEFAULT)) {
-          return true;
-        }
-        throw new IllegalStateException("Default namespace not found. Instead found unexpected namespaces: "
-                                          + namespaces);
+        // list() method will throw exception if the http request does not receive response
```
Comment (anwar6953) [inline cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java]: I think you should still have the comment
"Check that the dataset service is up", so its clear why we're calling `list()`

**Comment by anwar6953** [comment]
LGTM

**Comment by ecfm** [comment]
Build is successful and ready to merge
http://builds.cask.co/browse/CDAP-DUT4765-2

**Pull request merged**